odoo/tools/misc.py

```diff
@@ -1084,6 +1084,28 @@ def add(self, elem):
         OrderedSet.add(self, elem)
 
 
+class SimpleLRU(OrderedDict):
```

Comment:
Are the performances really so critical that we need a second LRU without synchronisation?

Reply:
Also while it might be the recipe from the stdlib I'm a bit wary of assuming all modifications will go through `__setitem__`. That's not the case of `dict` and it doesn't seem *officially* guaranteed by the `OrderedDict` documentation.

Reply:
I will compare performances with Odoo's existing `LRU`.
Also, the code of this class is copy-pasted from Python's official documentation, so I assume it to be correct.